backend/src/index.ts

```diff
@@ -1,30 +1,62 @@
-import bodyParser from 'body-parser'
-import express from 'express'
-import serveStatic from 'serve-static'
-import favicon from 'serve-favicon'
-import nocache from 'nocache'
-import { resolve, join } from 'path';
+import bodyParser from "body-parser";
+import express from "express";
+import serveStatic from "serve-static";
+import favicon from "serve-favicon";
+import nocache from "nocache";
+import { resolve, join } from "path";
+import dotenv from "dotenv-safe";
+dotenv.config();
 
-const app = express()
+type InjectedConfiguration = {
```

Comment:
Would be nice to share the types here in a `shared` folder (next to client / backend) but I believe the duplication is fine for the prototype.

Reply:
yup, I was thinking about adding that as well but I decided against it for now due to the increased complexity of the setup it will bring